src/main/java/net/earthcomputer/clientcommands/mixin/commands/fps/MinecraftMixin.java

```diff
@@ -0,0 +1,16 @@
+package net.earthcomputer.clientcommands.mixin.commands.fps;
+
+import com.llamalad7.mixinextras.injector.ModifyExpressionValue;
+import net.minecraft.client.Minecraft;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+
+@Mixin(Minecraft.class)
+public class MinecraftMixin {
+
+    @ModifyExpressionValue(method = "runTick", at = @At(value = "CONSTANT", args = "intValue=" + Options.UNLIMITED_FRAMERATE_CUTOFF))
```

Comment:
Just use `@ModifyConstant`. It is as bad as `@ModifyExpressionValue` if you don't use the original variable.

Reply:
`@ModifyConstant` conflicts with other mods, while `@MEV` does not.